pkg/validate/security_context.go

```diff
@@ -451,6 +451,24 @@ var _ = framework.KubeDescribe("Security Context", func() {
 			checkNetworkManagement(rc, containerID, notPrivileged)
 		})
 
+		It("selinux mount label should persist when container is privileged", func() {
```

Comment:
could you move this case to `selinux_linux.go`?

Reply:
Moved.